Source/WebKit/Shared/Cocoa/WebIconUtilities.mm

```diff
@@ -27,13 +27,21 @@
 #import "WebIconUtilities.h"
 
 #if PLATFORM(IOS_FAMILY)
-
 #import "UIKitSPI.h"
```

Comment:
This import should be moved underneath the `#if PLATFORM(COCOA)` since the rest of the file is wrapped in there.

But, do we even need this import? I don't see any obvious uses of SPI here. Let's remove it if possible.